website/docs/guides/getting-started.html.markdown

```diff
@@ -0,0 +1,592 @@
+---
+layout: "kubernetes"
+page_title: "Kubernetes: Getting Started with Kubernetes provider"
+sidebar_current: "docs-kubernetes-guide-getting-started"
+description: |-
+  This guide focuses on scheduling Kubernetes resources like Pods,
+  Replication Controllers, Services etc. on top of a properly configured
+  and running Kubernetes cluster.
+---
+
+# Getting Started with Kubernetes provider
+
+## Kubernetes
+
+[Kubernetes](https://kubernetes.io/) (K8S) is an open-source workload scheduler
+with focus on containerized applications.
+
+There are at least 2 steps involved in scheduling your first container
+on a K8s cluster. You need the K8S cluster with all its components
+running _somewhere_ and then schedule the K8S resources, like Pods,
+Replication Controllers, Services etc.
+
+This guide focuses mainly on the latter part and expects you to have
+a properly configured & running Kubernetes cluster.
+
+The guide also expects you to run the cluster on a cloud provider
+where K8S can automatically provision a load balancer.
+
+## Why Terraform?
+
+While you could use `kubectl` or similar CLI-based tools mapped to API calls
+to manage all K8S resources described in YAML files,
+orchestration with Terraform presents a few benefits.
+
+ - [HCL](https://www.terraform.io/docs/configuration/syntax.html) - same
+    language for both lower layer with underlying infrastructure (compute)
+    and scheduling layer
```

Comment:
I think someone not familiar with Terraform might have trouble understanding what the above means. I suggest a more general comment like:

> Use the same [configuration language](/docs/configuration/syntax.html) to provision the Kubernetes infrastructure and to deploy applications into it.

(Note also that our markdown style calls for us to use root-relative URLs for internal links.)